L1Trigger/L1CaloTrigger/interface/Phase2L1CaloEGammaUtils.h

```diff
@@ -1243,17 +1245,20 @@ namespace p2eg {
      * unique to each GCT card.
      */
     l1tp2::DigitizedClusterCorrelator createDigitizedClusterCorrelator(const int corrTowPhiOffset) const {
+      std::cout << " Creating digitized cluster correlator with towEta, crEta: " << towEta << ", " << crEta
+                << ", with global crystal iEta " << (towEta * CRYSTALS_IN_TOWER_ETA) + crEta << std::endl;
```

Comment:
Please remove the `std::cout` and change it to `edm::LogInfo` if necessary

Reply:
Yes, done in commit cc8a76697d8cb4c289cd179, should I also squash them into one commit?

Reply:
Thanks, removed the `std::cout` and squashed back into one commit.